gno.land/pkg/gnoweb/static/css/app.css

```diff
@@ -239,13 +239,22 @@ a[href="#"] {
   color: var(--muted-color, #757575);
 }
 
-ul {
+.gno-tmpl-section ul {
   padding: 0;
 }
 
-li {
+.gno-tmpl-section li ,
+#header li ,
+.footer li {
   list-style: none;
-  margin-bottom: 0.4rem;
+}
+
+.gno-tmpl-section blockquote {
+  margin-inline: 0;
+}
```

Comment:
gno-tmpl-section is unused

Reply:
it is used (on the homepage) 🤔

Reply:
No, thats gno-tmpl-page

Reply:
No 😛
<img width="1601" alt="Capture d’écran 2024-05-28 à 20 46 20" src="https://github.com/gnolang/gno/assets/7622257/6bc6067e-37b9-4b8f-a080-e6804eee1f1d">

Reply:
oh god we now have dynamically generated class names

we are going to regret this

whatever, resolving

Reply:
@thehowl No worry, it's just for short term.
But I would be curious to get your point of view on why this dynamic classes could be a bad thing in the future? For tests?

Reply:
it's a maintainability concern: if I have a function name, or a class name, I want to be able to use a global search to find all of its usages throughout the codebase, with nothing more complex than that or needing to have knowledge about a function that composes it

That way if we're refactoring the class name for some reason, we're not surprised in finding out it was being generated (when we couldn't even find it in the first place)

Furthermore, if it is just passed as a simple Kind, it actually allows the user of that struct to set whatever it wants, without selecting the value from an enum (a "safe" list of what we know, should work)

So, if anything, I would rather `Kind` was an enum, and the class name was generated in its entirety from gno, ie.

```
switch kind {
    case KindPage:
        return "gno-tmpl-page"
...
    default:
        panic("invalid kind")
}
```

Reply:
OK. So yeah I fully agree with you. To be honest, I started to create an enum to deal with this (as i'm used to do in Typescript) but I wanted to be sure the whole things works before implementing it. Also, I was looking for another way to deal with enum in go than using the switch I don't like (it is very verbose...). but maybe there is no other way to do it in go (Not a go expert).

Reply:
Sadly not. Lack of proper enums is my number 1 grievance in Go.

You can avoid a switch by using an array/slice (and numbering the enum sequentially from 0, using iota).